src/basics/strings/is-palindrom/index.js

```diff
@@ -5,6 +5,13 @@
  * isPalindrom('abcba') //=> true
  * isPalindrom('abcdba') //=> false
  */
-function isPalindrom(word) {}
+function isPalindrom(word) {
+    var reversed = word.split("").reverse().join("");
+    if (word == reversed) {
+        return true
+    }
+    return false
+}
+
```

Comment:
if we define `N` to be the length of `word`, what is the time and space complexity of this function?
Can we do better on space complexity?

Reply:
the complexity would be linear O(n)
I think we can't do better, cuz other methods use loops and they increase the complexity